layouts/_default/baseof.html

```diff
@@ -17,8 +17,11 @@
     {{ end }}
 
     {{ if $.Site.Params.enterprise_enabled }}
-      <link data-test="{{$basePath}}" rel="stylesheet" href="{{ path.Join $basePath "/presidium-enterprise.css" }}">
-      <link data-test="{{$basePath}}" rel="preload" href="{{ path.Join $basePath "/presidium-enterprise.js" }}" as="script">
+      {{ $enterpriseScript := $.Site.Params.enterprise_script | default "/presidium-enterprise.js" }}
```

Comment:
@Zalaras what is the need for this cahnge?

Reply:
@DustinFischer 
So by appending the version here it forces the browser to make a different call when there is a update. Basically a way to ignore a cache of an older version.

We were already setting the `enterprise_script` in the content builder just not using it in the theme

Content builder PR: https://github.com/SPANDigital/presidium-content-builder/pull/312

Reply:
@Zalaras can we not use Hugo fingerprinting for this? It feels a bit opaque that this is for cachebusting... 
Like, I'd keep this as is, but also see if you can fingerprint it...
Hugo is fantastic at cache busting

Reply:
https://gohugo.io/hugo-pipes/fingerprint/

Reply:
https://github.com/SPANDigital/presidium-layouts-base/pull/3#issuecomment-2479025471